mds/authent/generators.py

```diff
@@ -0,0 +1,46 @@
+from collections import namedtuple
+import uuid
+
+from django.utils import timezone
+import jwt
+
+from . import models
+from . import settings
+
+
+AuthMean = namedtuple("AuthMean", ("key", "algorithm"))
+AUTH_MEAN_RSA = AuthMean(settings.AUTHENT_RSA_PRIVATE_KEY, "RS256")
+AUTH_MEAN_SECRET = AuthMean(settings.AUTHENT_RSA_PRIVATE_KEY, "HS256")
+
+
+def generate_jwt(
```

Comment:
This function is only called for long-lived tokens ?
How does oauth2_provider generate the JWT (and its scopes) in a "normal" scenario ?